tools/@o3r/test-helpers/scripts/utilities/create-test-app.mjs

```diff
@@ -0,0 +1,113 @@
+import {execSync} from 'node:child_process';
+import {existsSync, mkdirSync, readFileSync, rmSync} from 'node:fs';
+import * as path from 'node:path';
+import {satisfies} from 'semver';
+import {Locker} from './locker.mjs';
+
+/**
+ * Generate a base app with minimal necessary dependencies
+ * Uses a locker mechanism so this function can be called in parallel
+ * The lock will automatically expire after 10 minutes if the creation of the app failed for whatever reason
+ *
+ * @param appName
+ */
+export async function createTestApp(options) {
+  options = {
+    appName: 'test-app',
+    cwd: process.cwd(),
+    globalFolderPath: process.cwd(),
+    registry: 'http://localhost:4873',
+    blank: false,
+    packageManager: 'yarn',
```

Comment:
As discussed, would help to be based on a env variable to be able to set a matrix of IT runs based on both package managers

Reply:
it is already handled by an env variable inside the tests

Reply:
If I did not miss something, today it is handle by a option passed to `prepare-test-app` cli.
I think a Env Var would be simpler

Reply:
updated, package manager can now be set using `ENFORCED_PACKAGE_MANAGER`